plugin/net/cni.go

```diff
@@ -203,15 +208,18 @@ func (c *CNIPlugin) CmdDel(args *skel.CmdArgs) error {
 		err = ipam.ExecDel(conf.IPAM.Type, args.StdinData)
 	}
 	// Hack - don't know how we should detect this situation properly
-	if args.Netns == "" && strings.Contains(err.Error(), "no addresses") {
-		err = nil
-	}
-	if err != nil {
-		return fmt.Errorf("unable to release IP address: %s", err)
+	if args.Netns != "" || !strings.Contains(err.Error(), "no addresses") {
```

Comment:
it's ok to log "no addresses"; the point of the code before was to not return an error back to kubelet, but you're never going to do that